tests/api/test_discourse.py

```diff
@@ -0,0 +1,14 @@
+import requests
+
+from actions.api.discourse import DiscourseAPI
+
+
+def test_get_discourse_links():
+    """Test that the link returned is correctly formatted and goes to a valid webpage"""
+    discourse = DiscourseAPI("https://forum.rasa.com/search")
+    disc_res = discourse.query("rasa")
+    disc_res = disc_res.json()
+    link_string = discourse.get_discourse_links(disc_res.get("topics"), 0) or ""
+    link = link_string.split("](")[1][:-1]
+    link_result = requests.get(link)
+    assert link_result.status_code == 200
```

Comment:
This will fail if the link_string is `""` right?

Reply:
Yes, this was a shortcut to avoid error when the return is `None`